src/components/List/List.styled.ts

```diff
@@ -1,10 +1,7 @@
 import styled from 'styled-components';
 
-export const ListContainer = styled.div`
-    flex: 1;
-    margin: 20px;
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: center;
-    align-items: center;
-`;
+export const ListContainer = styled.div<{isUser: boolean}>`
+    display: grid;
```

Comment:
Using grid display to get a more symmetric visual on the list, and allowing 5 items if it is team, and 4 items when its's an user (the Card becomes larger)